api/chains/models.go

```diff
@@ -170,3 +170,17 @@ type Pagination struct {
 type APRResponse struct {
 	APR float64 `json:"apr,omitempty"`
 }
+
+type ChainStatus struct {
+	Online bool `json:"online"`
+}
+
+type ChainsStatusesResponse struct {
+	Chains map[string]ChainStatus `json:"chains"`
+}
+
+func NewChainsStatusesResponse() ChainsStatusesResponse {
+	return ChainsStatusesResponse{
+		Chains: make(map[string]ChainStatus),
+	}
+}
```

Comment:
```suggestion
func NewChainsStatusesResponse(sz int) ChainsStatusesResponse {
	return ChainsStatusesResponse{
		Chains: make(map[string]ChainStatus, 0, sz),
	}
}
```
then on `GetChainsStatuses`  

```go
res := NewChainsStatusesResponse(len(status))
```

Reply:
It's a map so I omitted the "0" but added a size!

Reply:
opps my bad :p